pom.xml

```diff
@@ -723,12 +722,6 @@
                   </exclusion>
                 </exclusions>
             </dependency>
```

Comment:
We do not have restrict plugin in PQS. Should we add one here and sync all restricts or just add commons-collections restricts?

Reply:
It's up to you.
PQS has minimal code.
If you decide to add the restricions, then I think it's better to sync everything from the main repo.

Reply:
Yes make sense, let me raise another JIRA for same, this will keep changes in PR as per JIRA title

Reply:
Raised PHOENIX-7438